app/scripts/directives/downloadable_icon.js

```diff
@@ -5,31 +5,54 @@
  * @name designkitApp.directive:downloadableIcon
  * @restrict E
  * @param {String} src The URL of the SVG to be downloaded.
+ * @param {String} name The name (title) of the image.
  * @param {String} color The hex color (minus the leading #) to be applied to every `path`
  * within the SVG.
  * @description Download an SVG image and insert it into the document as a child of this
  * element. This then sets the `fill` attribute of all `path` elements to the color given,
- * and adds a data URI to download the icon in that color.
+ * and links to download the image with the new color as an SVG or a PNG.
  */
 angular.module('designkitApp').directive('downloadableIcon', function($http) {
   return {
     restrict: 'E',
     scope: {
       src: '@',
+      name: '@',
       color: '@'
     },
-    template: '<a></a>',
+    template: '<h3>{{ name }} <a>SVG</a> / <a>PNG</a></h3><div></div>',
```

Comment:
@allancorbett this is the template for the `downloadable-icon` directive. As long as the first `<a>` element is the SVG link, and the second is the PNG link, this should still work when you fix it.